Makefile

```diff
@@ -7,8 +7,8 @@ include $(shell go env GOPATH)/src/github.com/dedis/Coding/bin/Makefile.base
 # to `make test_playground`.
 test_playground:
 	for a in $$( seq 100 ); do \
-	  if DEBUG_TIME=true go test -v -race -short > log.txt 2>&1; then \
-		  echo Successfully ran at $$(date); \
+	  if DEBUG_TIME=true go test -v -race -short -run TestClient_SendProtobufParallel > log.txt 2>&1; then \
+		  echo Successfully ran \#$$a at $$(date); \
```

Comment:
please revert this part

Reply:
Done - although it's only for the `test_playground` target, which is only used when debug-testing with Travis. I kept the change of `\#$$a`, because this was missing...